Cargo.toml

```diff
@@ -88,6 +88,7 @@ members = [
     "tools/testdata-scripts",
     "tools/md-tests",
     "ffi/dart/tools/datagen",
+    "tools/graveyard",
```

Comment:
Suggestion: Maybe put this in the `exclude` section so that we don't touch `Cargo.lock` whenever someone does a graveyard update

Reply:
Then we can't inherit from the workspace 

That's fine I guess. I'll make it independent

Reply:
I think it is warranted to be explicit and independent.